### PR TITLE
[geometry] Add IDL tests for Geometry APIs

### PR DESCRIPTION
**Pull request by zcorpan**
This is on top of #5966.

Follows w3c/fxtf-drafts#167

### CODE REVIEW

**Comment by w3c-bots** [comment]
*This report has been truncated because the total content is 240601 characters in length, which is in excess of GitHub.com's limit for comments (65536 characters).

 [View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/233674851)

# Firefox (nightly) #
Testing web-platform-tests at revision a9646c18dcd03a9bad792c941c6a190a71e3c3cf
Ignoring 1 changed files:
Using browser at version BuildID 20170518100156; SourceStamp baf05f61bc14fdf45511bc1165ce76daa08c5c0f
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>2 tests ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5991/css/geometry-1/interfaces.html">/css/geometry-1/interfaces.html</a></summary>


|                                                                                                               Subtest                                                                                                                | Results |                                                                                                   Messages                                                                                                  |
|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
|                                                                                                                                                                                                                                      | OK      |                                                                                                                                                                                                             |
| `Test driver`                                                                                                                                                                                                                        | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: existence and properties of interface object`                                                                                                                                                           | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface object length`                                                                                                                                                                                           | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface object name`                                                                                                                                                                                             | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: existence and properties of interface prototype object`                                                                                                                                                 | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: existence and properties of interface prototype object's "constructor" property`                                                                                                                        | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: operation fromPoint(DOMPointInit)`                                                                                                                                                                      | FAIL    | `assert_own_property: interface object missing static operation expected property "fromPoint" missing`                                                                                                      |
| `DOMPointReadOnly interface: attribute x`                                                                                                                                                                                            | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: attribute y`                                                                                                                                                                                            | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: attribute z`                                                                                                                                                                                            | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: attribute w`                                                                                                                                                                                            | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: operation matrixTransform(DOMMatrixInit)`                                                                                                                                                               | FAIL    | `assert_own_property: interface prototype object missing non-static operation expected property "matrixTransform" missing`                                                                                  |
| `DOMPoint interface: existence and properties of interface object`                                                                                                                                                                   | PASS    |                                                                                                                                                                                                             |
| `DOMPoint interface object length`                                                                                                                                                                                                   | PASS    |                                                                                                                                                                                                             |
| `DOMPoint interface object name`                                                                                                                                                                                                     | PASS    |                                                                                                                                                                                                             |
| `DOMPoint interface: legacy window alias`                                                                                                                                                                                            | FAIL    | `assert_equals: self.SVGPoint should be the same value as self.DOMPoint expected function "function DOMPoint() {\n    [native code]\n}" but got function "function SVGPoint() {\n    [native code]\n}"`     |
| `DOMPoint interface: existence and properties of interface prototype object`                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMPoint interface: existence and properties of interface prototype object's "constructor" property`                                                                                                                                | PASS    |                                                                                                                                                                                                             |
| `DOMPoint interface: operation fromPoint(DOMPointInit)`                                                                                                                                                                              | FAIL    | `assert_own_property: interface object missing static operation expected property "fromPoint" missing`                                                                                                      |
| `DOMPoint interface: attribute x`                                                                                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMPoint interface: attribute y`                                                                                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMPoint interface: attribute z`                                                                                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMPoint interface: attribute w`                                                                                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMPoint interface: new DOMPoint() must inherit property "fromPoint" with the proper type (0)`                                                                                                                                      | PASS    |                                                                                                                                                                                                             |
| `DOMPoint interface: calling fromPoint(DOMPointInit) on new DOMPoint() with too few arguments must throw TypeError`                                                                                                                  | PASS    |                                                                                                                                                                                                             |
| `DOMPoint interface: new DOMPoint() must inherit property "x" with the proper type (1)`                                                                                                                                              | PASS    |                                                                                                                                                                                                             |
| `DOMPoint interface: new DOMPoint() must inherit property "y" with the proper type (2)`                                                                                                                                              | PASS    |                                                                                                                                                                                                             |
| `DOMPoint interface: new DOMPoint() must inherit property "z" with the proper type (3)`                                                                                                                                              | PASS    |                                                                                                                                                                                                             |
| `DOMPoint interface: new DOMPoint() must inherit property "w" with the proper type (4)`                                                                                                                                              | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: new DOMPoint() must inherit property "fromPoint" with the proper type (0)`                                                                                                                              | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: calling fromPoint(DOMPointInit) on new DOMPoint() with too few arguments must throw TypeError`                                                                                                          | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: new DOMPoint() must inherit property "x" with the proper type (1)`                                                                                                                                      | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: new DOMPoint() must inherit property "y" with the proper type (2)`                                                                                                                                      | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: new DOMPoint() must inherit property "z" with the proper type (3)`                                                                                                                                      | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: new DOMPoint() must inherit property "w" with the proper type (4)`                                                                                                                                      | PASS    |                                                                                                                                                                                                             |
| `DOMPointReadOnly interface: new DOMPoint() must inherit property "matrixTransform" with the proper type (5)`                                                                                                                        | FAIL    | `assert_inherits: property "matrixTransform" not found in prototype chain`                                                                                                                                  |
| `DOMPointReadOnly interface: calling matrixTransform(DOMMatrixInit) on new DOMPoint() with too few arguments must throw TypeError`                                                                                                   | FAIL    | `assert_inherits: property "matrixTransform" not found in prototype chain`                                                                                                                                  |
| `DOMRectReadOnly interface: existence and properties of interface object`                                                                                                                                                            | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface object length`                                                                                                                                                                                            | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface object name`                                                                                                                                                                                              | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: existence and properties of interface prototype object`                                                                                                                                                  | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: existence and properties of interface prototype object's "constructor" property`                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: operation fromRect(DOMRectInit)`                                                                                                                                                                         | FAIL    | `assert_own_property: interface object missing static operation expected property "fromRect" missing`                                                                                                       |
| `DOMRectReadOnly interface: attribute x`                                                                                                                                                                                             | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: attribute y`                                                                                                                                                                                             | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: attribute width`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: attribute height`                                                                                                                                                                                        | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: attribute top`                                                                                                                                                                                           | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: attribute right`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: attribute bottom`                                                                                                                                                                                        | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: attribute left`                                                                                                                                                                                          | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface: existence and properties of interface object`                                                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface object length`                                                                                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface object name`                                                                                                                                                                                                      | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface: legacy window alias`                                                                                                                                                                                             | FAIL    | `assert_equals: self.SVGRect should be the same value as self.DOMRect expected function "function DOMRect() {\n    [native code]\n}" but got function "function SVGRect() {\n    [native code]\n}"`         |
| `DOMRect interface: existence and properties of interface prototype object`                                                                                                                                                          | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface: existence and properties of interface prototype object's "constructor" property`                                                                                                                                 | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface: operation fromRect(DOMRectInit)`                                                                                                                                                                                 | FAIL    | `assert_own_property: interface object missing static operation expected property "fromRect" missing`                                                                                                       |
| `DOMRect interface: attribute x`                                                                                                                                                                                                     | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface: attribute y`                                                                                                                                                                                                     | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface: attribute width`                                                                                                                                                                                                 | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface: attribute height`                                                                                                                                                                                                | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface: new DOMRect() must inherit property "fromRect" with the proper type (0)`                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface: calling fromRect(DOMRectInit) on new DOMRect() with too few arguments must throw TypeError`                                                                                                                      | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface: new DOMRect() must inherit property "x" with the proper type (1)`                                                                                                                                                | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface: new DOMRect() must inherit property "y" with the proper type (2)`                                                                                                                                                | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface: new DOMRect() must inherit property "width" with the proper type (3)`                                                                                                                                            | PASS    |                                                                                                                                                                                                             |
| `DOMRect interface: new DOMRect() must inherit property "height" with the proper type (4)`                                                                                                                                           | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: new DOMRect() must inherit property "fromRect" with the proper type (0)`                                                                                                                                 | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: calling fromRect(DOMRectInit) on new DOMRect() with too few arguments must throw TypeError`                                                                                                              | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: new DOMRect() must inherit property "x" with the proper type (1)`                                                                                                                                        | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: new DOMRect() must inherit property "y" with the proper type (2)`                                                                                                                                        | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: new DOMRect() must inherit property "width" with the proper type (3)`                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: new DOMRect() must inherit property "height" with the proper type (4)`                                                                                                                                   | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: new DOMRect() must inherit property "top" with the proper type (5)`                                                                                                                                      | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: new DOMRect() must inherit property "right" with the proper type (6)`                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: new DOMRect() must inherit property "bottom" with the proper type (7)`                                                                                                                                   | PASS    |                                                                                                                                                                                                             |
| `DOMRectReadOnly interface: new DOMRect() must inherit property "left" with the proper type (8)`                                                                                                                                     | PASS    |                                                                                                                                                                                                             |
| `DOMQuad interface: existence and properties of interface object`                                                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMQuad interface object length`                                                                                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMQuad interface object name`                                                                                                                                                                                                      | PASS    |                                                                                                                                                                                                             |
| `DOMQuad interface: existence and properties of interface prototype object`                                                                                                                                                          | PASS    |                                                                                                                                                                                                             |
| `DOMQuad interface: existence and properties of interface prototype object's "constructor" property`                                                                                                                                 | PASS    |                                                                                                                                                                                                             |
| `DOMQuad interface: operation fromRect(DOMRectInit)`                                                                                                                                                                                 | FAIL    | `assert_own_property: interface object missing static operation expected property "fromRect" missing`                                                                                                       |
| `DOMQuad interface: operation fromQuad(DOMQuadInit)`                                                                                                                                                                                 | FAIL    | `assert_own_property: interface object missing static operation expected property "fromQuad" missing`                                                                                                       |
| `DOMQuad interface: attribute p1`                                                                                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMQuad interface: attribute p2`                                                                                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMQuad interface: attribute p3`                                                                                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMQuad interface: attribute p4`                                                                                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMQuad interface: operation getBounds()`                                                                                                                                                                                           | FAIL    | `assert_own_property: interface prototype object missing non-static operation expected property "getBounds" missing`                                                                                        |
| `DOMMatrixReadOnly interface: existence and properties of interface object`                                                                                                                                                          | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface object length`                                                                                                                                                                                          | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface object name`                                                                                                                                                                                            | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: existence and properties of interface prototype object`                                                                                                                                                | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: existence and properties of interface prototype object's "constructor" property`                                                                                                                       | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: operation fromMatrix(DOMMatrixInit)`                                                                                                                                                                   | FAIL    | `assert_own_property: interface object missing static operation expected property "fromMatrix" missing`                                                                                                     |
| `DOMMatrixReadOnly interface: operation fromFloat32Array(Float32Array)`                                                                                                                                                              | FAIL    | `assert_own_property: interface object missing static operation expected property "fromFloat32Array" missing`                                                                                               |
| `DOMMatrixReadOnly interface: operation fromFloat64Array(Float64Array)`                                                                                                                                                              | FAIL    | `assert_own_property: interface object missing static operation expected property "fromFloat64Array" missing`                                                                                               |
| `DOMMatrixReadOnly interface: attribute a`                                                                                                                                                                                           | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute b`                                                                                                                                                                                           | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute c`                                                                                                                                                                                           | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute d`                                                                                                                                                                                           | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute e`                                                                                                                                                                                           | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute f`                                                                                                                                                                                           | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m11`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m12`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m13`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m14`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m21`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m22`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m23`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m24`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m31`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m32`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m33`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m34`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m41`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m42`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m43`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute m44`                                                                                                                                                                                         | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute is2D`                                                                                                                                                                                        | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: attribute isIdentity`                                                                                                                                                                                  | FAIL    | `assert_true: The prototype object must have a property "isIdentity" expected true got false`                                                                                                               |
| `DOMMatrixReadOnly interface: operation translate(unrestricted double,unrestricted double,unrestricted double)`                                                                                                                      | FAIL    | `assert_equals: property has wrong .length expected 0 but got 2`                                                                                                                                            |
| `DOMMatrixReadOnly interface: operation scale(unrestricted double,unrestricted double,unrestricted double,unrestricted double,unrestricted double,unrestricted double)`                                                              | FAIL    | `assert_equals: property has wrong .length expected 0 but got 1`                                                                                                                                            |
| `DOMMatrixReadOnly interface: operation scale3d(unrestricted double,unrestricted double,unrestricted double,unrestricted double)`                                                                                                    | FAIL    | `assert_equals: property has wrong .length expected 0 but got 1`                                                                                                                                            |
| `DOMMatrixReadOnly interface: operation rotate(unrestricted double,unrestricted double,unrestricted double)`                                                                                                                         | FAIL    | `assert_equals: property has wrong .length expected 0 but got 1`                                                                                                                                            |
| `DOMMatrixReadOnly interface: operation rotateFromVector(unrestricted double,unrestricted double)`                                                                                                                                   | FAIL    | `assert_equals: property has wrong .length expected 0 but got 2`                                                                                                                                            |
| `DOMMatrixReadOnly interface: operation rotateAxisAngle(unrestricted double,unrestricted double,unrestricted double,unrestricted double)`                                                                                            | FAIL    | `assert_equals: property has wrong .length expected 0 but got 4`                                                                                                                                            |
| `DOMMatrixReadOnly interface: operation skewX(unrestricted double)`                                                                                                                                                                  | FAIL    | `assert_equals: property has wrong .length expected 0 but got 1`                                                                                                                                            |
| `DOMMatrixReadOnly interface: operation skewY(unrestricted double)`                                                                                                                                                                  | FAIL    | `assert_equals: property has wrong .length expected 0 but got 1`                                                                                                                                            |
| `DOMMatrixReadOnly interface: operation multiply(DOMMatrixInit)`                                                                                                                                                                     | FAIL    | `assert_equals: property has wrong .length expected 0 but got 1`                                                                                                                                            |
| `DOMMatrixReadOnly interface: operation flipX()`                                                                                                                                                                                     | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: operation flipY()`                                                                                                                                                                                     | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: operation inverse()`                                                                                                                                                                                   | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: operation transformPoint(DOMPointInit)`                                                                                                                                                                | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: operation toFloat32Array()`                                                                                                                                                                            | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: operation toFloat64Array()`                                                                                                                                                                            | PASS    |                                                                                                                                                                                                             |
| `DOMMatrixReadOnly interface: stringifier`                                                                                                                                                                                           | PASS    |                                                                                                                                                                                                             |
| `DOMMatrix interface: existence and properties of interface object`                                                                                                                                                                  | PASS    |                                                                                                                                                                                                             |
| `DOMMatrix interface object length`                                                                                                                                                                                                  | PASS    |                                                                                                                                                                                                             |
| `DOMMatrix interface object name`                                                                                                                                                                                                    | PASS    |                                                                                                                                                                                                             |
| `DOMMatrix interface: legacy window alias`

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/233674852)

# Sauce (safari) #
Testing web-platform-tests at revision a9646c18dcd03a9bad792c941c6a190a71e3c3cf
Ignoring 1 changed files:
Using browser at version 10.0
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>2 tests ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5991/css/geometry-1/interfaces.html">/css/geometry-1/interfaces.html</a></summary>


|    Subtest    | Results |           Messages           |
|---------------|---------|------------------------------|
|               | OK      |                              |
| `Test driver` | FAIL    | `Can't find variable: fetch` |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5991/css/geometry-1/interfaces.worker.html">/css/geometry-1/interfaces.worker.html</a></summary>


|    Subtest    | Results |           Messages           |
|---------------|---------|------------------------------|
|               | TIMEOUT |                              |
| `Test driver` | FAIL    | `Can't find variable: fetch` |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/233674858)

# Chrome (unstable) #
Testing web-platform-tests at revision a9646c18dcd03a9bad792c941c6a190a71e3c3cf
Ignoring 1 changed files:

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/233674859)

# Sauce (MicrosoftEdge) #
Testing web-platform-tests at revision a9646c18dcd03a9bad792c941c6a190a71e3c3cf
Ignoring 1 changed files:

File: css/geometry-1/support/interfaces.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    DOMQuad: ["new DOMQuad()"],
+    DOMMatrixReadOnly: ["new DOMMatrixReadOnly()", "DOMMatrixReadOnly.fromMatrix({is2D: false})"],
+    DOMMatrix: ["new DOMMatrix()", "DOMMatrix.fromMatrix({is2D: false})"],
+  });
+  idlArray.test();
+  done();
+}
+
+promise_test(function() {
+  return fetch("/interfaces/geometry.idl").then(response => response.text())
```
Comment (tobie) [inline css/geometry-1/support/interfaces.js]: `fetch` doesn't seem to work in Saucelab's Safari. Do we care?

**Review by tobie — APPROVED** [review]
I guess you'll want to remove the diff for idlharness.js before merging. Other than that, LGTM.

File: css/geometry-1/support/interfaces.js (hunk 1 above)
Comment (rwaldron) [inline css/geometry-1/support/interfaces.js]: @tobie I cannot answer that question, but I can offer that webrtc tests are also using fetch: https://github.com/w3c/web-platform-tests/blob/master/webrtc/interfaces.html#L51

Comment (zcorpan) [inline css/geometry-1/support/interfaces.js]: Don't think so... I copied this from another IDL test, and `fetch` works in Safari TP.

**Pull request merged**